projects/plugins/jetpack/modules/custom-css/custom-css/preprocessors/lessc.inc.php

```diff
@@ -1301,6 +1304,7 @@ public function assertArgs($value, $expectedArgs, $name="") {
 	}
 
 	protected function toHSL($color) {
+		$H = null;
```

Comment:
Really the thing to do would be to change the `elseif ($b == $max)` on line 1328 to just `else`. After doing `$max = max( $r, $g, $b )`, if the result is not equal to `$r` or `$g` then either it's equal to `$b`, or someone snuck a NaN in, or there's some really weird floating point error going on.